src/lyra/core/types.py

```diff
@@ -79,6 +81,75 @@ def __repr__(self):
         return f"List[{repr(self.typ)}]"
 
 
+class TupleLyraType(LyraType):
+    """Tuple type representation."""
+    # TODO: maybe support tuples of variable length, esp. empty tuple?:
```

Comment:
No, variable length tuple types are not yet supported. But they are maybe not really needed for our purposes, since as far as I know they are only used for star arguments.